### PR TITLE
Removed pre-Hawthorn support from tox, webpack test fix

### PR DESCRIPTION
**Pull request by johnbaldwin**
Tests were breaking and backward compatibility is impeding forward
development

Fixed webpack loader issue with Figures UI app view breaking

### CODE REVIEW

**Comment by thraxil** [comment]
@johnbaldwin does this meant that figures is dropping ficus/ginkgo support altogether?

File: devsite/devsite/test_settings.py [hunk 1]
```diff
@@ -106,17 +103,20 @@ def root(*args):
...
+    'FIGURES_APP': {
+        'BUNDLE_DIR_NAME': 'figures/',
+        'STATS_FILE': 'tests/test-webpack-stats.json',
+    }
+}
 CELERYBEAT_SCHEDULE = {}
 FEATURES = {}
 
 # Declare values we need from server vars (e.g. lms.env.json)
 ENV_TOKENS = {
```
Comment (OmarIthawi) [inline devsite/devsite/test_settings.py]: Remove the commented code.

```suggestion
ENV_TOKENS = {}
```

**Comment by johnbaldwin** [comment]
@thraxil As you know, it is an opportunity cost. I want to drop Ficus/Ginkgo support unless there is a critical business need to backport fixes to Ficus/Ginkgo instead of move our customers forward to Hawthorn+. We have unmaintained branches for Ficus/Ginkgo if needed

**Comment by johnbaldwin** [comment]
So, we are going to support Ginkgo from a special Ginkgo branch we rework from master

**Pull request merged**